sphinx/source/governance.rst

```diff
@@ -94,3 +108,19 @@ At any stage during the voting process and before the proposal is completed, the
 
 This means future votes will be ignored, and the proposal will never be accepted. However it will remain visible as a proposal so members can easily audit historic proposals.
 
+Updating trusted enclave code versions
+--------------------------------------
+
+For new nodes to be able to join the network, the version of the code they run (as specified by the ``--enclave-file``) should be first trusted by the consortium of members.
+
+If the version of the code being executed needs to be updated (for example, to support additional JSON-RPC endpoints), members can create a ``new_code`` proposal, specifying the new code version (e.g. ``3175971c02d00c1a8f9dd23ca89e64955c5caa94e24f4a3a0579dcfb2e6aebf9``):
+
+.. code-block:: bash
+
+    memberclient --cert member_cert --privk member_privk --rpc-address node_ip:node_port --ca network_cert add_code --new-code-id code_version
+
+Once the proposal has been accepted, nodes running the new code are authorised join the network. This allows stopping nodes running older versions of the code.
```

Comment:
Nodes running older versions of the code can be removed from the network and stopped. It's the removal that allows them to be stopped [without causing consensus to stop], not the addition of the new nodes as such.